### DISCUSSION TITLE
Chromedriver 115

### DISCUSSION THREAD

**Discussion by nlehrer** [giggio/node-chromedriver#419] [Q&A] [answered]
Hi, I just want to check if there is an expected timeline for releasing chromedriver 115 in this package. From the README I see some discussion of configuring custom binaries for it, but if I am understanding correctly the actual version 115 hasn't been released? I ask because the stable Chrome version is now 115 so my workflow involving `DETECT_CHROMEDRIVER_VERSION` is broken.

**Comment by giggio** [comment, marked as answer]
I was waiting for the stable to release to become available. I just released it today.

**Reply by steeples-sans** [reply]
Where can it be found?

**Reply by giggio** [reply]
On npm.
